hordes-ui-mod.js

```diff
@@ -18,36 +18,45 @@
   * TODO: (MAYBE, Confirm if dev is ok w it) Ctrl clicking item in inventory when Merchant is open to paste item name into search field, and search
   * TODO: (MAYBE, confirm if dev is ok w it) Ctrl clicking item to copy details so user can paste in chat
   */
-(function() {
+(function () {
     'use strict';
 
     // If this version is different from the user's stored state,
     // e.g. they have upgraded the version of this script and there are breaking changes,
     // then their stored state will be deleted.
     const BREAKING_VERSION = 1;
-    const VERSION = '0.190'; // Should match version in UserScript description
+    const VERSION = '0.200'; // Should match version in UserScript description
 
     const DEFAULT_CHAT_TAB_NAME = 'Untitled';
     const STORAGE_STATE_KEY = 'hordesio-uimodsakaiyo-state';
 
     let state = {
-    	breakingVersion: BREAKING_VERSION,
-    	chat: {
-    		GM: true,
-    	},
-    	windowsPos: {},
-    	blockList: {},
-    	friendsList: {},
-    	mapOpacity: 70, // e.g. 70 = opacity: 0.7
-    	friendNotes: {},
-    	chatTabs: [],
+        breakingVersion: BREAKING_VERSION,
+        chat: {
+            GM: true,
+        },
+        windowsPos: {},
+        blockList: {},
+        friendsList: {},
+        mapOpacity: 70, // e.g. 70 = opacity: 0.7
+        friendNotes: {},
+        chatTabs: [],
+        xpMeterState: {
+            currentXp: 0,
+            xpArr: [],
+            averageXp: 0,
+            gainedXp: 0,
+            currentLvl: 0
+        }
     };
+
     // tempState is saved only between page refreshes.
     const tempState = {
-    	// The last name clicked in chat
-    	chatName: null,
-    	lastMapWidth: 0,
-    	lastMapHeight: 0,
+        // The last name clicked in chat
+        chatName: null,
+        lastMapWidth: 0,
+        lastMapHeight: 0,
+        xpMeterInterval: null,
```

Comment:
Can you add a comment above this line, explaining briefly what `xpMeterInterval` is used for?